### ISSUE TITLE
Current /about/ and /about/partners/ are horrible hacks

### ISSUE THREAD

**Issue by rossbruniges** [mozilla/make.mozilla.org#24]
So very soon /about/ will not have the content that it has now - which is currently a page about the summer campaign.

So basically we should change it to be more like this:

/about/ - for now redirects to /about/summer-code-party-2012/
/about/summer-code-party-2012/ - (summer-code-party-2012 is the slug) which can then link to a static page of our choosing
/partners/ - throws a 404
/partners/summer-code-party-2012/ - (summer-code-party-2012 is the campaign slug) which displays all partners related to that campaign

Does this make sense and give you stuff to work with?

**Comment by fidothe** [comment]
How about:

```
/about/
/about/summer-code-party-2012
/about/summer-code-party-2012/partners
```

Obv, when I say 'how about?' I mean 'I'm doing this' ;-) It's basically just a urls.py change, with a bit of S&R in templates for the about path.

**Comment by rossbruniges** [comment]
I thought that - but to me the three level thing sounded a bit lame - and we're not really talking 'about' the campaign partners - just listing them...

**Comment by rossbruniges** [comment]
GitHub interface on iPhone is difficult!

**Comment by fidothe** [comment]
Fixed in 478ccd870ebd3b5208b8c81f267be4eb3535fc42
